cmd/cli/kubectl-kyverno/commands/apply/command.go

```diff
@@ -67,8 +67,7 @@ type ApplyCommandConfig struct {
 }
 
 // allow os.exit to be overwritten during unit tests
-var osExit = os.Exit
-
+// var osExit = os.Exit
```

Comment:
Is `osExit` used anywhere other than in the unit tests? Do we still need it?

Reply:
No we dont need it, kept it as a comment only.
Should I remove it completely?

Reply:
Yes, we should have no commented code.